### PR TITLE
add black check workflow

### PR DESCRIPTION
**Pull request by epassaro**
## Description

A simple action to run `black --check tardis` on every push to a PR.

## Motivation and Context

Enforce PEP8.

## How Has This Been Tested?
- [x] Testing pipeline
- [ ] Other (please describe)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] None of the above (please describe)

## Checklist:

- [x] My code follows the code style of this project
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have built the documentation on my fork following [these instructions](https://tardis-sn.github.io/tardis/development/documentation_preview.html)
- [x] I have assigned and requested two reviewers for this pull request

### CODE REVIEW

**Comment by epassaro** [comment]
This is ready to merge from my point of view.

I'm not going to blackify the files because I don't know so much about `black`. If you want to make a PR to this PR would be fine. @wkerzendorf @jaladh-singhal.

File: .github/workflows/black-check.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+
+      - name: Install Black
+        run: pip install black
+
+      - name: Run Black
+        run: black --check tardis
```
Comment (jaladh-singhal) [inline .github/workflows/black-check.yml:26]: Looks very good! Please make sure that this command is taking black configuration file we have at tardis in account. If it is not, then you can also explicitly point it out by `--config` flag. Details here: https://black.readthedocs.io/en/stable/pyproject_toml.html#where-black-looks-for-the-file

**Review by jaladh-singhal — COMMENTED** [review]
@epassaro thank you for making this - it was very necessary!

File: .github/workflows/black-check.yml (hunk 1 above)
Comment (epassaro) [reply .github/workflows/black-check.yml:26]: Yes, it taking config from `pyproject.toml` just right.

**Review by jaladh-singhal — APPROVED** [review]

**Pull request merged**